src/ScribuntoLuaLibrary.php

```diff
@@ -37,29 +37,30 @@ public function register() {
 	 *
 	 * @return array
 	 */
-	public function getQueryResult( $queryString = null ) {
```

Comment:
At the top (after the namespace) you need to add something like `use SMWQueryProcessor as QueryProcessor`;

```
..S....SPHP Fatal error:  Class 'SMW\Scribunto\QueryProcessor' not found in /home/travis/build/SemanticMediaWiki/mw/extensions/SemanticScribunto/src/ScribuntoLuaLibrary.php on line 44

PHP Stack trace:

```